### PR TITLE
* Updated query and config files to implement  the participants table

### PR DESCRIPTION
**Pull request by Senait7-ux**

### CODE REVIEW

**Comment by Senait7-ux** [comment]
https://github.com/CBIIT/bento-c3dc-frontend/pull/8

File: src/bento/dashboardTabData.js [hunk 1]
```diff
@@ -438,75 +438,78 @@ query sampleOverview(
...
-        phs_accession
-        study_id
-        race
-        gender
-        ethnicity
-        alternate_participant_id
-    }
-}
-`;
+export const GET_PARTICIPANTS_OVERVIEW_QUERY = gql`query participantOverview(
```
Comment (iksheth) [inline src/bento/dashboardTabData.js:441]: @Senait7-ux can you move "query participantOverview" to next line?

**Review by iksheth — CHANGES_REQUESTED** [review]

File: src/bento/dashboardTabData.js (hunk 1 above)
Comment (Senait7-ux) [reply src/bento/dashboardTabData.js]: Updated

**Review by iksheth — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**